### ISSUE TITLE
Crash when starting RetopoFlow in specific scene

### ISSUE THREAD

**Issue by jlampel** [CGCookie/retopoflow#1127]
We got a bug report from a user via email. His scene reliably crashes RetopoFlow. I've narrowed it down somewhat, but I haven't been able to find which specific object is causing the issue. Hiding or disabling all of them will solve the problem but deleting them one by one yields confusing results since it doesn't seem to matter which one gets deleted. 

[Tubu Sculpt_crash_02.zip](https://github.com/CGCookie/retopoflow/files/8827853/Tubu.Sculpt_crash_02.zip)

**Comment by bryanf3000** [comment]
Yeah, Retopoflow is crashing blender every time I try to use the conture tool.

**Comment by jlampel** [comment]
This actually still hard crashes v3, but v4 has no problem with it. @jfranmatheu Can you take a look when you have a chance?
